all_words_gen/src/inflection_info.rs

```diff
@@ -24,7 +24,9 @@ pub fn create_inflection_infos(
     };
 
     for stem_info in stem_infos {
-        let infs = igen.generate_all_inflections(&stem_info.pali1);
+        let mut infs = igen.generate_all_inflections(&stem_info.pali1);
+        infs.sort();
+        infs.dedup();
```

Comment:
not the right fix.

this is too early in the process to be removing duplicates. instead we need as much info in the all_words db as possible.

lets retain all the (inflection, pali1), meaning we'll have to mark them as unique by adding another column